### ISSUE TITLE
Status page: Show information about supported image types by GD

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#5760]
See: https://www.drupal.org/project/drupal/issues/3116611

We currently show this:
![image](https://user-images.githubusercontent.com/2423362/188265605-09b90cbe-667b-43cc-81e4-d5b1e0101ac3.png)

In Drupal, they currently have this:
![image](https://user-images.githubusercontent.com/2423362/188265616-9ac9dd90-55ec-4785-8302-e2b9ae583236.png)

And the suggestion for the new UI is this:
![image](https://user-images.githubusercontent.com/2423362/188265642-44afc42f-3ef8-4b06-99dc-c54e9f692044.png)

**Comment by klonos** [comment]
Before:
![image](https://user-images.githubusercontent.com/2423362/188265605-09b90cbe-667b-43cc-81e4-d5b1e0101ac3.png)

PR up for review: https://github.com/backdrop/backdrop/pull/4188

Everything OK (details collapsed by default - expanded in this screenshot for preview purposes):
![image](https://user-images.githubusercontent.com/2423362/188267819-dbf2ea9f-ec32-424c-b704-ce4994c605a5.png)

Image formats supported - effects not supported:
![image](https://user-images.githubusercontent.com/2423362/188267951-7174e9cc-ad1d-4b48-90ca-0bde5ced8d6e.png)

Support for one image type missing:
![image](https://user-images.githubusercontent.com/2423362/188268377-89a16181-ad6f-490e-9356-24d8a8261fd1.png)

Support for more than one image types missing (`format_plural()` instead of plain `t()`):
![image](https://user-images.githubusercontent.com/2423362/188268420-06d959ad-9f0e-415a-bd05-502298110db4.png)

Only one image format supported:
![image](https://user-images.githubusercontent.com/2423362/188268833-c5b81f45-bafe-4ec2-b668-245178c857ec.png)

No image formats supported - effects still supported (odd, but may happen):
![image](https://user-images.githubusercontent.com/2423362/188268857-eff01607-9f39-46d4-a799-62bf5a5e02ef.png)

Problematic GD installation/support:
![image](https://user-images.githubusercontent.com/2423362/188268543-49f397eb-a52a-468f-b230-089c60f569b0.png)

**Comment by klonos** [comment]
...looking at these screenshots, the most important information (the image formats) seems to be at the end of the sentence, in a way that they are not easily scannable. So I have pushed an alternative PR here: https://github.com/backdrop/backdrop/pull/4189

![image](https://user-images.githubusercontent.com/2423362/188279290-9ba61d91-e801-4b07-8b28-deb6d6bd4840.png)

- Lists each image format and effect individually, with their name at the start of the bullet point, which improves the readability/scannability of the list.
- Replaces the default bullet points with green ticks (✓) and red x's (✗) next to supported/problematic formats and effects which helps sighted people focus on the problematic ones, while at the same time retains the "supported"/"not supported" text bits of text for people that use screen readers.
- Splits the informational text (the instructions on how to fix things) to the bottom of the list of the image effects/formats, and adds titles to both sections, which organizes things a bit better + also helps with accessibility.

Checked CSS to make sure it works on RTL as well:

**Comment by klonos** [comment]
Thank you @olafgrabienski for the feedback on the PR:

>Under the heading "Supported image formats and effects" may be listed supported and unsupported things. I'd suggest to replace "Supported" with "Support of" in the heading: `Support of image formats and effects`

Yeah, you are right, but I didn't really like either of these options now 😅 ...I've played around with some other ideas, and here's what I ended up with:
![image](https://user-images.githubusercontent.com/2423362/194730120-8c1da321-de45-41fc-af48-e6c7a822f435.png)
![image](https://user-images.githubusercontent.com/2423362/194730128-c735d6f4-c830-4808-abac-a9702a245e62.png)

What do you think? What do others think?

**Comment by klonos** [comment]
...also thank you @indigoxela 🙏🏼 ...I've addressed your concerns/feedback re accessibility in the PR.

**Comment by klonos** [comment]
> If I understand https://github.com/backdrop/backdrop/commit/e752124b2be7b94b158ac87c49bcead32ebbd12b correctly, ...

Yes, that's correct @olafgrabienski 👍🏼 ...that's what I ended up using in order to get the PR out there, but I have to admit that I'm not 100% happy with the "All" thing in `All image formats and effects supported`. What I wanted to communicate is "all image effects/formats supported by core are supported by php and GD" but that is too wordy - saying "all" image formats/effects might be misperceived as "everything that's out there in the world", which is not true.

I have asked for feedback and better suggestions from the people in the weekly dev meeting, so I'll wait for that.

**Comment by indigoxela** [comment]
@klonos this looks great now! One thing concerns me a bit, though: is the lack of support for WebP an actual problem, worth a "warning" status?

People on older PHP versions probably won't have support in their GD, but that only matters if they allow webp extension in (field) uploads. Per default core doesn't require support for that format. If it's there - nice. If not - no big problem.

Please don't get me wrong: I totally support the improved display on the status page. I'm just not sure about the "warning" state.

![image-info](https://user-images.githubusercontent.com/842066/195838918-3a7b3181-8108-445c-b193-4d1a112fb91a.png)
